.github/workflows/releaseNovoElements.yml

```diff
@@ -0,0 +1,44 @@
+name: Release Novo Elements
+on:
+  workflow_dispatch:
+    inputs:
+    type:
+      description: 'Update Type (patch, major, minor)'
```

Comment:
Not sure exactly how this gets consumed, but just for standardization, should this be:

`major, minor, patch`?

Reply:
That bit is just a hint for for the options available in the action.   What is input gets used by `npm version` to bump the version number in the package.json and create the tag.   Updated the order, thank you